lib/rucio/web/rest/flaskapi/accountapp.wsgi

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+import sys
+import logging
+logging.basicConfig(stream=sys.stderr)
+sys.path.insert(0, '/opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/flask')
```

Comment:
Seems like a bad idea to hardcode absolute paths in the code?

Reply:
Yes, I know. It's just a first commit so that I can run a test build with readthedocs. There is still a lot to do on this and a there will be a lot more commits coming including some cleanups before this can be merged.

Reply:
Gotcha - will mark it as a WIP branch.

However - it might be reasonable to try and break it up into as small of PRs as possible... with >1,400 lines changed, it's already fairly difficult...